ui/src/components/gallery/QasGallery.stories.js

```diff
@@ -0,0 +1,87 @@
+import QasGallery from './QasGallery.vue'
+
+const noSummary = {
+  type: { summary: null }
+}
+
+export default {
+  component: QasGallery,
+  title: 'Components/Gallery',
+
+  parameters: {
+    docs: {
+      description: {
+        component: 'Gallery component, with list of images and carousel.'
+      }
+    }
+  },
+
+  argTypes: {
+    // Props
+    carouselNextIcon: {
+      description: 'Define a next image icon for carousel.'
+    },
+
+    carouselPrevIcon: {
+      description: 'Define a previous image icon for carousel.'
+    },
```

Comment:
Não seria `define *the* next`?

Reply:
Corrigido tanto para o next quanto para o previous.